dataplane/unittest/meson.build

```diff
@@ -9,7 +9,11 @@ dependencies += dependency('gtest_main')
 sources = files('unittest.cpp',
                 'ip_address.cpp',
                 'hashtable.cpp',
-                'sdp.cpp')
+                'sdp.cpp',
+                'random_prefixes.cpp',
+                'vrf_lpm_linear.cpp',
+                '../vrf.cpp',
+                )
```

Comment:
This should be done like in `controlplane/unittest/meson.build`, with sources defined in a separate variable:
```meson
dataplane_sources = files('../vrf.cpp',
                         )

sources = files('unittest.cpp',
                'ip_address.cpp',
                'hashtable.cpp',
                'sdp.cpp',
                'random_prefixes.cpp',
                'vrf_lpm_linear.cpp',
                'vrf_lpm_binary.cpp',
                )

arch = 'corei7'
cpp_args_append = ['-march=' + arch]

unittest = executable('yanet-dataplane-unittest',
                      [dataplane_sources, sources],
```